### ISSUE TITLE
Quickstart guide linked in the documentation is a dead link

### ISSUE THREAD

**Issue by JoleneRenee21** [internetarchive/openlibrary#9318]
### Problem

There is a link to the [Quickstart](https://github.com/internetarchive/openlibrary/wiki/Getting-Started) guide in both the README and CONTRIBUTING documentation that no longer works - it redirects to "Create new page" for the wiki. I have located the [Outdated Getting Started](https://github.com/internetarchive/openlibrary/wiki/OUTDATED-Getting-Started#quick-start-for-open-librarys-developers) wiki page which I believe was previously the destination of the documentation link. Since much of the content on this guide is outdated, I would recommend changing the documentation to point to [Start Here](https://github.com/internetarchive/openlibrary/wiki/1_Start-Here) and/or [Common Setup Tasks](https://github.com/internetarchive/openlibrary/wiki/1_Start-Here) as a temporary replacement for some of the quick information, such as admin login credentials and frequently needed docker commands. These were the best replacements I could find in the wiki for the not outdated info that may be helpful. I can make these changes to the documentation with confirmation that these would be appropriate replacements or given another alternative.

#### Evidence / Screenshot
![image](https://github.com/internetarchive/openlibrary/assets/100251491/96e22944-1d54-45ef-a822-9cd80b366622)

#### Relevant URL(s)
[README](https://github.com/internetarchive/openlibrary?tab=readme-ov-file#developers-guide)
[CONTRIBUTING](https://github.com/internetarchive/openlibrary/blob/master/CONTRIBUTING.md#installing-open-library)
[Quickstart](https://github.com/internetarchive/openlibrary/wiki/Getting-Started)

### Reproducing the bug

1. Go to the [README](https://github.com/internetarchive/openlibrary?tab=readme-ov-file#developers-guide) or [CONTRIBUTING](https://github.com/internetarchive/openlibrary?tab=readme-ov-file#developers-guide) files.
2. Locate the [Quickstart](https://github.com/internetarchive/openlibrary/wiki/Getting-Started) link and click on it.

* Expected behavior: Opening a wiki page with quick start information.
* Actual behavior: Redirect to create a new page for the wiki called Getting Started.

### Context

- Browser (Chrome, Safari, Firefox, etc): Reproduced on Firefox, Chrome, and Edge
- OS (Windows, Mac, etc): Windows
- Logged in (Y/N): Y (to Github)
- Environment (prod, dev, local): N/A

**Comment by Asudox** [comment]
Looking at the latest archive of the quickstart guide, there was a text mentioning how outdated it was and that it has to be rewritten. They most likely removed it for the time being as the guide gets rewritten.

https://web.archive.org/web/20240324115306/https://github.com/internetarchive/openlibrary/wiki/Getting-Started

**Comment by rebecca-shoptaw** [comment]
@Asudox Yes, there is an onboarding overhaul in progress, but it would still be good to point these links to [Start Here](https://github.com/internetarchive/openlibrary/wiki/1_Start-Here) for now so that new contributors don't get confused!

**Comment by AbhinavKRN** [comment]
@rebecca-shoptaw is this issue available?

**Comment by rebecca-shoptaw** [comment]
@AbhinavKRN I believe @JoleneRenee21 is still working on it -- we're waiting on a final decision re: where the link should go instead.

**Comment by JoleneRenee21** [comment]
Yes, just waiting on the decision and then will update again :)

**Comment by rebecca-shoptaw** [comment]
@JoleneRenee21 sounds good! will double-check to see if any progress has been made on that front! :)

**Comment by AbhinavKRN** [comment]
@rebecca-shoptaw any update?

**Comment by rebecca-shoptaw** [comment]
@AbhinavKRN To clarify, the issue is not available, we're waiting on an update to get it finished. There should be plenty of other good first issues available or opening up soon if you'd like to find something to work on 🙂

**Comment by AbhinavKRN** [comment]
@rebecca-shoptaw Acutally i have commented on some, can you please check it and reply. It would be a great help.
